build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,109 @@
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "NodePortLocal" "default" true) }}
+
+# Enable Antrea ClusterNetworkPolicy feature to complement K8s NetworkPolicy for cluster admins
+# to define security policies which apply to the entire cluster, and Antrea NetworkPolicy
+# feature that supports priorities, rule actions and externalEntities in the future.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "AntreaPolicy" "default" true) }}
+
+# Enable flowexporter which exports polled conntrack connections as IPFIX flow records from each
+# agent to a configured collector.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "FlowExporter" "default" false) }}
+
+# Enable collecting and exposing NetworkPolicy statistics.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "NetworkPolicyStats" "default" true) }}
+
+# Enable controlling SNAT IPs of Pod egress traffic.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Egress" "default" true) }}
+
+# Enable AntreaIPAM, which can allocate IP addresses from IPPools. AntreaIPAM is required by the
+# bridging mode and allocates IPs to Pods in bridging mode. It is also required to use Antrea for
+# IPAM when configuring secondary network interfaces with Multus.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "AntreaIPAM" "default" false) }}
+
+# Enable multicast traffic.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Multicast" "default" false) }}
+
+# Enable Antrea Multi-cluster Gateway to support cross-cluster traffic.
+# This feature is supported only with encap mode.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Multicluster" "default" false) }}
+
+# Enable support for provisioning secondary network interfaces for Pods (using
+# Pod annotations). At the moment, Antrea can only create secondary network
+# interfaces using SR-IOV VFs on baremetal Nodes.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "SecondaryNetwork" "default" false) }}
+
+# Enable managing external IPs of Services of LoadBalancer type.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "ServiceExternalIP" "default" false) }}
+
+# Enable mirroring or redirecting the traffic Pods send or receive.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "TrafficControl" "default" false) }}
+
+# Enable certificated-based authentication for IPsec.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "IPsecCertAuth" "default" false) }}
+
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-int1.
+Secondary_OVSBridge1: {{ (.Values.secondaryNetworkOVS).bridgeName1 | quote }}
```

Comment:
I think we should just name it OVSIntegrationBridge. It is already in secondary-network.conf, no need for the "Secondary_" prefix.

Reply:
Aligned and made the required changes.